crates/red_knot_python_semantic/src/types.rs

```diff
@@ -362,6 +364,10 @@ impl<'db> Type<'db> {
                 // TODO defer to Type::Instance(<bytes from typeshed>).member
                 Type::Unknown
             }
+            Type::Tuple(_) => {
+                // TODO: defer to `typing.Tuple` / `builtins.tuple` methods from typeshed's stubs
```

Comment:
[`typing.Tuple`](https://github.com/python/typeshed/blob/65170f4a2ef8dc81d3678624ac2a4610d1c6045e/stdlib/typing.pyi#L210) won't tell us much here ;) all the interesting stuff is in `builtins.pyi`

```suggestion
                // TODO: defer to `builtins.tuple` methods from typeshed's stubs
```

Reply:
Oh, thanks, that makes sense.